server/utils/utils.ts

```diff
@@ -238,6 +238,9 @@ export const generateListMetadata = (
   }
 }
 
+export const formatCurrency = (number: number, currency: string): string =>
```

Comment:
There is already a `formatMoney` helper - is this needed in addition to that?

Reply:
Yes the currency helper is required. It matches the format of the existing profile when values are 4 figures and above. Here is an example on dev - /prisoner/G6123VU/sentence-and-release scroll down to sentence 4